sumologic/util.go

```diff
@@ -296,3 +302,112 @@ func GetTimeRangeBoundary(tfRangeBoundary map[string]interface{}) interface{} {
 
 	return nil
 }
+
+/*
+This function returns a function (in 2 variants) determining whether two stringe representation
+of a time value can be considered equivalent. One variant compares only absolute values (ignores '-' sign)
+and the other compare relative values. For details see util_test.go::TestSuppressTimeDiff.
+
+Some examples (we accept time units for seconds, minutes, hours, days and weeks):
+-1h = -60m
+1h20m = 80m
+60m60m60m1h = 3h30m30m
+-60m = 1h (only if we compare absolute values, so with isRelative = false)
```

Comment:
@MZochowskiSUMO , Is user allowed to set isRelative field ? why are we allowing it if there is no difference if user enters -ve / +ve value

Reply:
@rmamilla-sumo No it is not available to users. isRelative is only a flag we use in code (to suppress diffs) to specify if we treat -ve / +ve values as the same value. In most places there are equivalent, but here afaik they are not: https://github.com/SumoLogic/terraform-provider-sumologic/pull/442/files#diff-4bc5e99d3cf4cbda61368cb6fdbafe1bc97884202cca74d578e12b053064cb25R176, https://github.com/SumoLogic/terraform-provider-sumologic/pull/442/files#diff-b749eb8ae5017ef0ed2545e094d7e38321f015a91b335265140abcf75fb61365R503 (relativeTimeRangeSchema). Can you confirm that?

Reply:
@rmamilla-sumo We are suppressing time diff values treating -ve/+ve as equal everywhere except the 2 places above. isRelative flag only changes the semantics of diff suppression, it does not enable/disable it.

Reply:
I changed suppression to absolute (isRelative = false) in SLO monitors' trigger_conditions.